src/traces/parcoords/plot.js

```diff
@@ -0,0 +1,51 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var parcoords = require('./parcoords');
+var Lib = require('../../lib');
+
+module.exports = function plot(gd, cdparcoords) {
+
+    var fullLayout = gd._fullLayout;
+    var svg = fullLayout._paper;
+    var root = fullLayout._paperdiv;
+    var data = cdparcoords.map(function(d, i) {
```

Comment:
Great. And while doing, can you make `parcoord.js` expect simply the `cdparcoords` (i.e. the `parcoords` calcdata traces) instead of this new `data` array that has nothing special about? This should make the code easier to read.